src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPage.tsx

```diff
@@ -45,8 +45,12 @@ const ActionDetailsPage = () => {
 
   const isMotion = action?.isMotion;
   const createdAt = action?.createdAt;
+  const isInvalidMotion = isMotion && !action.motionData;
   const isInvalidTransaction =
-    isInvalidTransactionHash || isUnknownTransaction || !action;
+    isInvalidTransactionHash ||
+    isUnknownTransaction ||
+    !action ||
+    isInvalidMotion;
```

Comment:
Most of these checks (at least the tx) one is a legacy, simple, check from back in the dapp days. It's basically a static string format check.

I'm not saying now, but in the future, this is good place we can refactor in order to have more smart checks (we can even check them via a block ingestor directly on chain, and keep a record of them, with various metadata, in the database)

Reply:
https://github.com/JoinColony/colonyCDapp/issues/347